### PR TITLE
Role permissions: separate cluster initialisation from db initialisation

### PR DESCRIPTION
**Pull request by m-kuhn**
Untangles

 - role creation - one shot cluster initialization jobs
 - permissions - that need to be granted once per database

### CODE REVIEW

File: 12_1_roles.sql [hunk 1]
```diff
@@ -1,15 +1,6 @@
...
-      END IF;
-    END LOOP;
-END
-$$;
+------------------------------------------
+/* GRANT on schemas - once per database */
+------------------------------------------
 
 /* Viewer */
 GRANT USAGE ON SCHEMA qgep_od  TO qgep_viewer;
```
Comment (m-kuhn) [inline 12_1_roles.sql:6]: @ponceta are the permissions on the sequences required for a viewer or can that be granted for user only?

**Review by 3nids — APPROVED** [review]

File: 12_1_roles.sql (hunk 1 above)
Comment (sjib) [reply 12_1_roles.sql:6]: Everything what is needed to run the calculations for labelling or additional attributes such as _*functionhierarchic has to be granted, else we will have wrong labellings in the view I guess.

Comment (m-kuhn) [reply 12_1_roles.sql:6]: Do we need sequences for this?

Comment (ponceta) [reply 12_1_roles.sql:6]: To my knowledge it is not required, but I rarely use the qgep_viewer role.

**Pull request merged**